lib/tasks/demo.rake

```diff
@@ -36,17 +36,49 @@ namespace :demo do
   desc 'Generate N many more monitorees based on existing data'
   task create_bulk_data: :environment do
```

Comment:
might be out of scope for this PR, but thoughts on doing something like this for duplication? https://stackoverflow.com/questions/2783150/mysql-how-to-copy-rows-but-change-a-few-fields

it would definitely require a bit of work, but I think the potential for performance gains would be massive

Reply:
That is a good idea but I also think it would be best in another PR